io/zenoh-transport/tests/unicast_intermittent.rs

```diff
@@ -408,3 +408,15 @@ fn transport_ws_intermittent() {
     let endpoint: EndPoint = format!("ws/127.0.0.1:{}", 12010).parse().unwrap();
     task::block_on(transport_intermittent(&endpoint));
 }
+
+#[cfg(feature = "transport_shm")]
+#[test]
```

Comment:
I believe there is no need to run this test every time on the CI (it will make increase its duration).
Mark the test as follows:
```
#[test]
#[ignore]
```
It can be then run with `cargo test -- --ignored`.
Please apply the changes to `transport_ws` as well.